crypto/mod.ts

```diff
@@ -81,6 +83,7 @@
  * [w3c/webcrypto#270](https://github.com/w3c/webcrypto/issues/270)), but until
  * that time, `timingSafeEqual()` is provided:
```

Comment:
problem with this line and the entire section above it is that it will be part of the example; would need further reorganizing, with then descriptive example titles